po/fr.po

```diff
@@ -30,25 +30,25 @@ msgid "-----------output-range:-----------"
 msgstr "---------échelle_de_sortie:--------"
 
 msgid "------selectable_dimensions(pix):------"
-msgstr "-------dimensions_a_choisir(pix):------"
+msgstr "-------dimensions_sélectionnable(pix):------"
```

Comment:
This needs to be the same width as the string above. Easiest option is to remove some of the "-" characters.